src/components/app/details/cicdHistory/TriggerDetails.tsx

```diff
@@ -131,9 +143,10 @@ const WorkerStatus = React.memo(({ message, podStatus, stage }: WorkerStatusType
 })
 
 const ProgressingStatus = React.memo(
-    ({ status, message, podStatus, stage, type }: ProgressingStatusType): JSX.Element => {
+    ({ status, startedOn, message, podStatus, stage, type }: ProgressingStatusType): JSX.Element => {
```

Comment:
Type is missing here

Reply:
Yes I have given the type in ProgressingStatusType.